core/runtime/src/main/java/io/quarkus/logging/LoggingFilter.java

```diff
@@ -0,0 +1,23 @@
+package io.quarkus.logging;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Makes the filter class known to Quarkus by the specified name.
+ * The filter can then be configured for a handler (like the logging handler using {@code quarkus.log.console.filter}).
+ *
+ * This class must ONLY be placed on implementations of {@link java.util.logging.Filter} that are marked as {@code final}.
+ */
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.TYPE)
+public @interface LoggingFilter {
+
+    /**
+     * Name with which the filter is referred to in configuration
+     */
+    String name();
```

Comment:
Should this have a default value? The FQN perhaps?

Reply:
I believe it's best to not have a value, that way it's very clear what the user has to configure in `application.properties`. At least that's the intention :)
If it does not work out like that in practice, we can always relax